### PR TITLE
Revert E-puck name to `e-puck`

### PR DESCRIPTION
**Pull request by ygoumaz**
#574 fixed the name from `e-puck` to `epuck` in the rats life example.

To be consistent with the default name of the E-puck, this PR reverts the change and modifies the name from `epuck` to `e-puck` in the launcher and the epuck_world instead.

### CODE REVIEW

File: webots_ros2_epuck/worlds/epuck_world.wbt [hunk 1]
```diff
@@ -52,7 +52,7 @@ WoodenBox {
 E-puck {
   translation 0.0530459 0.00806403 0.001
   rotation 0 0 1 3.1415
-  name "epuck"
+  name "e-puck"
```
Comment (omichel) [inline webots_ros2_epuck/worlds/epuck_world.wbt:55]: That's the default name, no need to specify it.
```suggestion
```

**Review by omichel — APPROVED** [review]
Did you test that the service names published by ROS are working fine with the dash in the robot name?

**Comment by ad-daniel** [comment]
In theory names are sanitized here: https://github.com/cyberbotics/webots_ros2/blob/db38ed564910a08413169c308013a6b726913653/webots_ros2_driver/src/Driver.cpp#L39-L41

**Comment by ygoumaz** [comment]
Yes I tested and it works fine. Dashes are replaced by underscores.

**Pull request merged**